manifests/apache.pp

```diff
@@ -123,4 +123,6 @@
     group   => 'root',
     mode    => '0644',
   }
+  
+  selinux::boolean { 'httpd_use_openstack': }
```

Comment:
We're not using a module for selinux. That means we could use ```selboolean``` which is built in, but then you need to be sure that selinux is enabled (either enforcing or permissive).

It's also only needed for crane so it belongs in ```pulp::crane::apache```.